### ISSUE TITLE
Auto Width no Longer Works in React

### ISSUE THREAD

**Issue by tnoetzel** [nhn/tui.chart#595]
**Describe the bug**
Auto width no longer works in React... getting error `To use auto layout, the width or height of the container must be specified as a value such as "%" or "vh", "vw"` even though width is specified.

**To Reproduce**
```
 const chartOptions = {
        chart: {
            height: 500,
            width: 'auto',
            title: 'Monthly Revenue',
            format: '1,000'
        },
        ...
}

...

<ColumnChart
    style={{ width: "100%" }}
    ref={chartRef}
    data={chartData}
    options={chartOptions}
/>
```

**Expected behavior**
Expected to work as in documentation... should get a responsive chart. 

**Desktop (please complete the following information):**
 - OS: all
 - Browser: all
 - Version: 4.1.1

**Comment by tnoetzel** [comment]
Minimum reproducible example here: https://codesandbox.io/s/toast-ui-responsive-bug-minimal-example-old3c?file=/src/index.js

**Comment by jung-han** [comment]
@tnoetzel It will be released after fixed it today. I'm sorry!

**Comment by tnoetzel** [comment]
@jung-han - When should I expect a new version on NPM?

**Comment by jung-han** [comment]
@tnoetzel 
I will release it today. When released, I will let you know by comment.

**Comment by jung-han** [comment]
@tnoetzel 
Resolved in [4.1.2](https://github.com/nhn/tui.chart/releases/tag/v4.1.2)! Check this version!

**Comment by tnoetzel** [comment]
> @tnoetzel
> Resolved in [4.1.2](https://github.com/nhn/tui.chart/releases/tag/v4.1.2)! Check this version!

Works perfectly! Thank you!

**Comment by pinakipatrapro** [comment]
Hi Experts, 
I am using the  **@version 4.2.1**   and is still able to reproduce the same issue in heatmap chart

const options = {
			**chart: {
				title: '24-hr Average Temperature', width: 'auto',
				height: 'auto'
			},**
			xAxis: {
				title: 'Month',
			},
			yAxis: {
				title: 'City',
			},
			tooltip: {
				formatter: (value) => `${value}°C`,
			},
			legend: {
				align: 'bottom',
			},
			series: {
				dataLabels: {
					visible: true,
				},
			},
			exportMenu: {
				visible: false
			},
			responsive: {
				animation: { duration: 300 }
			}

		};

		const chart = toastui.Chart.heatmapChart({ el, data, options });
